content/posts/2025-03-10-deepseek-r1.md

```diff
@@ -0,0 +1,221 @@
+---
+date: '2025-03-09T09:30:00-08:00'
+draft: false
+title: 'DeepSeek-R1 671B multi-host Deployment in AIBrix'
+disableShare: true
+hideSummary: true
+searchHidden: false
+ShowReadingTime: false
+ShowWordCount: false
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+ShowRssButtonInSectionTermList: false
+UseHugoToc: true
+ShowToc: true
+tocopen: true
+---
+
+This blog post introduce deploying DeepSeek R1 using AIBrix. DeepSeek-R1 demonstrates remarkable proficiency in reasoning tasks through step-by-step training process.It features 671B total parameters with 37B active parameters, and 128k context length. However, due to its 671B model size, the deployment process is a bit more complicated. AIBrix provides enough tools that enables users to deploy and manage distributed inference services efficiently.
```

Comment:
line 18  
"introduce" => "introduces"
"However, due to its 671B model size, the deployment process is a bit more complicated." => "However, due to its large size, the deployment process is more complex."

Reply:
addressed